libraries/.foodcritic

```diff
@@ -0,0 +1,4 @@
+~FC011
```

Comment:
Well, it's not exactly the *wrong directory*, but running on cookbooks would solve the problem (but this is a workaround), or maybe we should just add an `exclude_path` if we don't want libraries to be analyzed by it.